hadoop-hdfs-project/hadoop-hdfs/src/main/java/org/apache/hadoop/hdfs/server/namenode/FSDirectory.java

```diff
@@ -1091,6 +1091,24 @@ static void unprotectedUpdateCount(INodesInPath inodesInPath,
     }
   }
 
+  /**
+   * check that all parent directories have quotas set.
+   */
+  static boolean verifyIsQuota(INodesInPath iip, int pos) {
```

Comment:
looks like pos is always assigned iip.length() -1. So why not simplify the logic and remove the pos?

Reply:
the method name verifyIsQuota() isn't intuitive.